plugins/cost-insights/src/index.ts

```diff
@@ -18,7 +18,8 @@ export { plugin } from './plugin';
 export * from './client';
 export * from './api';
 export * from './components';
-export { useCurrency } from './hooks';
+export { useCurrency, CurrencyContext } from './hooks';
```

Comment:
What do we need to export `CurrencyContext` for?

Reply:
@ryanmvazquez We need it to mock the currency provider when using `CostGrowth` internally in GKE alerts.

Reply:
Just for testing? We already export `MockCurrencyContext` in `./utils/tests`